charts/atlantis/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: v0.16.1
 description: A Helm chart for Atlantis https://www.runatlantis.io
 name: atlantis
-version: 3.12.11
+version: 3.12.12-procore-0.0.1
```

do we want to keep 3.12.11 if we are versioning our changes separately? 
```suggestion
version: 3.12.11-procore-0.0.1
```

---

I did it this way because of how semantic versioning orders "pre-releases". Our "pre-release" is ahead of 3.12.11.
`3.12.11 < 3.12.12-procore-0.0.1 < 3.12.12`

---

ok makes sense!